src/components/HomePage/ReadingSection/NewCard/index.tsx

```diff
@@ -32,23 +32,23 @@ const NewCard: React.FC = () => {
     <>
       <Card
         onClick={() => {
-          logButtonClick('homepage_grow_beyond_ramadan');
+          logButtonClick('homepage_best_day');
         }}
         className={styles.firstTimeReadingCard}
-        link={getBeyondRamadanNavigationUrl()}
+        link={getBestDayNavigationUrl()}
         isNewTab
       >
         <div className={styles.cardOuterContainer}>
           <div className={styles.cardWithIcon}>
-            <div className={styles.iconContainer}>🚀</div>
+            <div className={styles.iconContainer}>🕋</div>
             <div className={styles.link}>
               <Trans
-                i18nKey="home:grow-beyond-ramadan"
+                i18nKey="home:best-day"
                 components={{
                   modalLink: (
                     <Link
                       variant={LinkVariant.Blend}
-                      href={getBeyondRamadanNavigationUrl()}
+                      href={getBestDayNavigationUrl()}
                       className={styles.linkHref}
                       onClick={onShareQuranClicked}
```

Comment:
the handler of clicking should be changed from `onShareQuranClicked` to `onBestDayClicked` and also the event inside should be changed.